### PR TITLE
Sean controllers

### PR DESCRIPTION
**Pull request by seanyles**

### CODE REVIEW

File: Development/NutriMon/src/main/java/chillingMonsters/NutriMonMain.java [hunk 1]
```diff
@@ -1,11 +1,28 @@
...
+import chillingMonsters.Controllers.NutriMonController;
+import chillingMonsters.Controllers.StockController;
 import chillingMonsters.Pages.loginPage;
 import static javafx.application.Application.*;
 
 public class NutriMonMain {
 
     public static void main(String[] args) {
-        launch(loginPage.class, args);
+      StockController stock = ControllerFactory.makeStockController();
```
Comment (mavalos1) [inline Development/NutriMon/src/main/java/chillingMonsters/NutriMonMain.java]: Change this back to normal launch

File: Development/NutriMon/src/main/resources/config.properties [hunk 2]
```diff
@@ -1,4 +1,4 @@
-dbpassword = 11111111
+dbpassword = root
```
Comment (mavalos1) [inline Development/NutriMon/src/main/resources/config.properties]: Undo this change (different local password)

**Review by mavalos1 — CHANGES_REQUESTED** [review]

**Review by mavalos1 — APPROVED** [review]

File: Development/NutriMon/src/main/resources/config.properties (hunk 2 above)
Comment (zonghaoliu) [reply Development/NutriMon/src/main/resources/config.properties]: This is fine though
